packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -642,6 +642,8 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
               </section>
             )}
 
+            <ExtensionSlot state={{ patientUuid }} name="extra-visit-attribute-slot" />
```

Comment:
Thanks @ODORA0 , lets make this name a bit more intuitive because its related to billing